packages/common-all/src/dnode.ts

```diff
@@ -1082,29 +1082,49 @@ export class NoteUtils {
   }
 
   /**
-   * Add {@link DNodeProps["parent"]}, {@link DNodeProps["children"]}, and backlinks from {@link noteHydrated} to {@link noteRaw}
+   * Add derived metadata from `noteHydrated` to `noteRaw`
+   * By default, include the following properties:
+   *  - parent
+   *  - children
+   *  - backLinks
```

Comment:
Would default include just parent/children since we have to explicitly keep backlinks?

Reply:
your right - updated